ftplugin/ps1.vim

```diff
@@ -11,6 +11,8 @@ if exists("b:did_ftplugin") | finish | endif
 " Don't load another plug-in for this buffer
 let b:did_ftplugin = 1
 
+compiler psake
```

Comment:
I'm inclined to agree. Technically the compiler doesn't do anything unless
invoked by the user, but I can see how having it there could be potentially
confusing if the user is using a regular Makefile to drive their process
and suddenly it tries to invoke Psake/Pester. So you think just having the
compilers available and letting the user choose with :compiler psake would
be more friendly?

On Wed, Jun 29, 2016 at 6:00 PM, Heath Stewart <notifications@github.com>
wrote:

> In ftplugin/ps1.vim
> <https://github.com/PProvost/vim-ps1/pull/17#discussion_r69043818>:
>
> > @@ -11,6 +11,8 @@ if exists("b:did_ftplugin") | finish | endif
> >  " Don't load another plug-in for this buffer
> >  let b:did_ftplugin = 1
> >
> > +compiler psake
>
> Quite a few assumptions are made here about the script, so I don't think
> we should default the compiler. By far, I would expect the majority of
> scripts to be just normal scripts psake would fail to execute. Besides,
> @PProvost <https://github.com/PProvost> has generally pushed back on
> putting any general user preferences in these files (and I generally agree)
> and let them define how they want them set. We could maybe add tips to the
> README.md, but even that may be outside the purview of this project.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PProvost/vim-ps1/pull/17/files/7d520788b1337da4010839a87c94e73a3fed1784#r69043818>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAdxXk0NbPNUVwc92yKLubg2lMjFlfClks5qQvkfgaJpZM4JBZbW>
> .
>